Dockerfile

```diff
@@ -211,11 +211,6 @@ RUN apk --no-cache add git openssh; \
  # Enable Xdebug
  docker-php-ext-enable xdebug
 
-# For Xdebuger to work, it needs the docker host ID
-# - in Mac AND Windows, `host.docker.internal` resolve to Docker host IP
-# - in Linux, `172.17.0.1` is the host IP
-ENV XDEBUG_CLIENT_HOST="host.docker.internal"
```

Comment:
I am thinking of keeping the Environment Variable reference in PHP Config, as well as its default value in Dockerfile. This makes the image decoupled from the docker-compose file. So, just in case you wish not to use the already existing `docker-compose.yaml` file, you won't need to change anything the Dockerfile to make it "configurable".

Reply:
Thanks for detailed explanation. I have recovered the environment variable.